spec/std/time/time_spec.cr

```diff
@@ -745,6 +745,29 @@ describe Time do
     Time.days_in_year(1990).should eq(365)
   end
 
+  it "knows which years are leap_year?" do
+    { 1900 => false,
```

Comment:
Great to include a spec for this!

Though I'd suggest to write it like this:
```crystal
{1900, 1965, 1999, 2001, 2018, 2019, 2021, 2099, 2100, 2101, 2200}.each do |year|
  Time.leap_year?(year).should be_false
end
{1968, 1972, 2000, 2004, 2020, 2400}.each do |year|
  Time.leap_year?(year).should be_true
end
```
I'd prefer this but it's just a nitpicking suggestion...

Reply:
Assuming that you also meant to change those to be valid syntax for `Array(Int32)` with `[…]` rather than `{…}`, I'd be OK with that. Let me know if you intended that to be an action item, too, or _just a nitpick_.

Reply:
I think it would be better that way, but I don't know what others think...
`{ ... }` was intended, that's a tuple literal.

Reply:
Ah, very new to crystal so that's my first encounter with a tuple literal!

Reply:
It might also be quite reasonable to split this into 4 separate test cases:
* 400-year leap years: `{1600, 2000, 2400}`
* 100-year normal years: `{1700, 1800, 1900, 2100, 2200, 2300}`
* typical non-century leap years: `{1968, 1972, 2004, 2020}`
* a smattering of non-century normal years: `{1965, 1999, 2001, 2018, 2019, 2021, 2099, 2101}`